src/AUTHORS

```diff
@@ -17,3 +17,4 @@ Robert Escriva <robert@hyperdex.org>
 # PebblesDB authors:
 Pandian Raju <pandian@cs.utexas.edu>
 Vijay Chidambaram <vijay@cs.utexas.edu>
+Zeyuan Hu <iamzeyuanhu@utexas.edu>
```

Comment:
Could you also add Jayashree Mohan (jaya@cs.utexas.edu)

Reply:
done